app.test.js

```diff
@@ -0,0 +1,24 @@
+
+import { describe, expect, it } from 'vitest'
+// import app from '/server.js';
```

Comment:
No comment in the code, unless you have good reason for the comment.

Reply:
done